.github/workflows/release_pipeline.yml

```
@@ -0,0 +1,320 @@
+      success: ${{ steps.release.outputs.success }}
+      version: ${{ needs.detect-changes.outputs.core-version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+          token: ${{ secrets.GH_TOKEN }}
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)